lib/classes/map_coordinates.rb

```diff
@@ -0,0 +1,44 @@
+class MapCoordinates
+  def initialize(location)
+    @location = location.gsub(" ", "%20")
```

Comment:
I'd recommend leaving this as is, and then wrapping your `api_url` with `URI.encode("https://...")`. That'll convert spaces into `%20` as well as all the other things you haven't thought of